src/jest/config.json

```diff
@@ -13,7 +13,8 @@
     "!ui_framework/components/**/*/index.js"
   ],
   "moduleNameMapper": {
-    "^ui_framework/components": "<rootDir>/ui_framework/components"
+    "^ui_framework/components": "<rootDir>/ui_framework/components",
+    "\\.(css|less|scss)$": "<rootDir>/src/jest/mockCssFiles.js"
```

Comment:
Sure, we can start simple and add as time goes on. I'm just not sure I see a problem with having a comprehensive list up front. These files will never actually be useful within a Jest test, right? So adding them all now will solve all problems we'll encounter with them, for everyone, forever.

On the other hand, if we kept this list simple to start with, then everyone who adds a new static file will go through the same painful process (which I went through):

1. They'll get an obscure webpack error about a missing loader
2. Do a Google search for 20 mins
3. Figure out it's a problem with static files
4. Probably end up just pasting in the comprehensive list of static files, since it's the recommendation on the Jest website (see link in PR description)

All of that, for what gain?